.eslintrc.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
```

Comment:
if you can get this to be marked as a file move that would be helpful to preserve blame/history :)

Reply:
will look into how

Reply:
Spoke with Tim about this offline, we determined it wasn't a blocker